src/main/java/ladder/Ladder.java

```diff
@@ -20,6 +21,24 @@ public Ladder(List<Line> lines) {
         this.lines = lines;
     }
 
+    public LadderResult run(Players players, InputResult inputResult) {
+        HashMap<String, String> ladderResult = new HashMap<>();
+
+        for (int startPoint = 0; startPoint < players.size(); startPoint++) {
+            int endPoint = runPerPerson(startPoint);
+            ladderResult.put(players.get(startPoint), inputResult.get(endPoint));
+        }
+        return new LadderResult(ladderResult);
+    }
+
+    int runPerPerson(int startPoint) {
```

Comment:
```suggestion
    private int runPerPerson(int startPoint) {
```
요구사항 구현 깔끔하게 해주셨네요 💯